relay-server/src/actors/envelopes.rs

```diff
@@ -1520,6 +1520,19 @@ impl EnvelopeProcessor {
             }
 
             event._metrics = Annotated::new(metrics);
+
+            if event.ty.value() == Some(&EventType::Transaction) {
+                let source = event
+                    .transaction_info
+                    .value()
+                    .and_then(|info| info.source.value())
+                    .unwrap_or(&TransactionSource::Unknown);
+
+                metric!(
+                    counter(RelayCounters::EventTransactionSource) += 1,
+                    source = &source.to_string()
```

Comment:
Do we want to tag this by platform and/or SDK version?

Reply:
This is a great suggestion, going to add this before merging.